lib/Jsrt/Jsrt.cpp

```diff
@@ -1215,6 +1215,12 @@ CHAKRA_API JsStringToPointer(_In_ JsValueRef stringValue, _Outptr_result_buffer_
 
 CHAKRA_API JsConvertValueToString(_In_ JsValueRef value, _Out_ JsValueRef *result)
 {
+    if (value != nullptr && Js::JavascriptString::Is(value))
```

Comment:
i'd think we need to also ensure that ScriptContext for value is the same as the current context?

Reply:
done.